src/command/check/check.c

```diff
@@ -101,7 +101,7 @@ checkStandby(const DbGetResult dbGroup, const unsigned int pgPathDefinedTotal)
         dbFree(dbGroup.standby);
     }
     // If backup from standby is true then warn when a standby not found
-    else if (cfgOptionBool(cfgOptBackupStandby))
+    else if (cfgOptionStrId(cfgOptBackupStandby) != CFGOPTVAL_BACKUP_STANDBY_N)
     {
         LOG_WARN("option '" CFGOPT_BACKUP_STANDBY "' is enabled but standby is not properly configured");
```

Comment:
Should we emit this warning if backup-standby=prefer? :thinking: 
By setting "prefer", the user accepts that there's no standby reachable, but in the other ends it's good to know that there's no standby available. It's just a small question, I don't have a strong opinion on this :shrug:

Reply:
I think check should definitely be warning here. Even when using prefer backing up from the primary is not ideal so we want to let the user know it will happen.

Reply:
Okay, good for me. I just wanted to make sure we thought about it :-)